tetrateci/setup_boring_go.sh

```diff
@@ -14,12 +14,12 @@ if $(grep -q "1.10" <<< $TAG); then
   export GOLANG_VERSION=1.16.9b7
 fi
 
-if $(grep -q "1.11" <<< $TAG); then
+if $(grep -q "1.11" <<< $TAG || grep -q "1.12" <<< $TAG); then
```

Comment:
If you do this, it won't enter the other if below with version 1.12, right? Shouldn't it be one or the other?